<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html><head><title>Truyen Tran</title>

<meta content="en-us" http-equiv="Content-Language">
<meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
<meta http-equiv="Content-type" content="text/html;charset=UTF-8">
<link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Abel">
<style type="text/css">
/* Layout-provided Styles */
ul.itemize {
margin-top: 0.7ex;
margin-bottom: 0.7ex;
margin-left: 3ex;
text-align: left;
}
</style>
<style>
body {
font-family: 'Abel', serif;
font-size: 12px;
}
</style></head>
<body>
<table style="border-collapse: collapse; width: 1038px; height: 1304px;" id="1" border="0" bordercolor="#111111" cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td style="border-right: 1px solid;" v="" bgcolor="#000000">&nbsp;</td>
<td>&nbsp;</td>
<td style="vertical-align: top;" v="">&nbsp;</td>
</tr>
<tr>
<td rowspan="2" v="" style="border-width: 1px; border-right: 1px solid; vertical-align: top;">
<p align="right"><img style="border: 2px solid ; width: 200px; height: 200px;" alt="portrait" src="figs/14064962%20390x390%20jpg.jpg" hspace="0"><br>
(Source: <a href="http://www.teksystems.com/resources/teksavvy-blog/2013/august/healthcare-analytics-present-opportunities-poses-challenges">TEKsystems</a>)</p>
<p align="right">
</p>
<p align="right">
</p>
<p align="right">
</p>
<p align="right"><font size="5"><a href="index.html">Home</a></font>&nbsp;
&nbsp; </p>
<br>
<p align="justify"></p>
<p align="justify">&nbsp;</p>
<p align="justify"></p>
<p align="justify">&nbsp;</p>
<p align="justify">&nbsp;</p>
</td>
</tr>
<tr>
<td width="24">
<p></p>
<p>&nbsp;</p>
<p>&nbsp;</p>
</td>
<td width="837">
<p>
<table style="width: 100%; text-align: left; margin-left: auto; margin-right: auto;" border="0" bordercolor="#000000" cellpadding="15" cellspacing="3">
<tbody>
<tr>
<td style="background-color: white;">
<p><font style="font-weight: bold; color: rgb(0, 102, 0);" size="+3">Deep learning for biomedicine</font></p><p><font style="color: rgb(0, 102, 0);" size="+3">A tutorial @ACML17, Seoul, Nov 2017.</font><font style="font-weight: bold; color: rgb(0, 102, 0);" size="+3"></font></p><p style="color: black; font-weight: bold; text-decoration: underline;"><font size="+2">Slides</font></p><br><ul style="color: black;">
</ul>
</td>
</tr>
</tbody>
</table>
</p>
<hr><br>
<p align="justify"><font style="font-weight: bold;" size="5">References</font><font size="5"><span style="text-decoration: underline;"></span></font><br></p><span style="font-weight: bold;">Genomics &amp; drug design<br></span><ol start="1"><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Duvenaud, David K., et al. "<a href="http://papers.nips.cc/paper/5954-convolutional-networks-on-graphs-for-learning-molecular-fingerprints">Convolutional networks on graphs for learning molecular fingerprints</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">Advances in neural information processing systems</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">. 2015.</span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"></span><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Romero, Adriana, et al. "<a href="https://arxiv.org/abs/1611.09340">Diet Networks: Thin Parameters for Fat Genomic</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">arXiv preprint arXiv:1611.09340</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"><span>&nbsp;</span>(2016).</span><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"></span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Stoiber, Marcus, and James Brown. "<a href="https://www.biorxiv.org/content/early/2017/05/01/133058">BasecRAWller: Streaming Nanopore Basecalling Directly from Raw Signal</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">bioRxiv</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"><span>&nbsp;</span>(2017): 133058.</span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Teng, Haotien, et al. "<a href="https://www.biorxiv.org/content/early/2017/08/23/179531">Chiron: Translating nanopore raw signal directly into nucleotide sequence using deep learning</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">bioRxiv</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">(2017): 179531.</span></li><li><font size="5">Boža, Vladimír, Broňa Brejová, and Tomáš Vinař. "<a href="http://journals.plos.org/plosone/article?id=10.1371/journal.pone.0178751">DeepNano: Deep recurrent neural networks for base calling in MinION nanopore reads</a>." <span style="font-style: italic;">PloS one</span> 12.6 (2017): e0178751.</font></li><li><font size="5">Alipanahi, Babak, et al. "<a href="http://www.nature.com/nbt/journal/v33/n8/full/nbt.3300.html">Predicting the sequence specificities of DNA-and RNA-binding proteins by deep learning</a>." <span style="font-style: italic;">Nature biotechnology</span> 33.8 (2015): 831-838.</font></li><li><font size="5">Trang Pham, Truyen Tran, Hoa Dam, Svetha Venkatesh, <a href="https://arxiv.org/abs/1708.04357"><span style="text-decoration: underline;">Graph Classification via Deep Learning with Virtual Nodes</span></a> <span style="font-style: italic;">Third Representation Learning for Graphs Workshop (ReLiG 2017)</span>.</font></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Gilmer, Justin, et al. "<a href="https://arxiv.org/abs/1704.01212">Neural message passing for quantum chemistry</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">arXiv preprint arXiv:1704.01212</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"><span>&nbsp;</span>(2017).</span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Ching, Travers, et al. "<a href="https://www.biorxiv.org/content/early/2017/05/28/142760">Opportunities And Obstacles For Deep Learning In Biology And Medicine</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">bioRxiv</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"><span>&nbsp;</span>(2017): 142760.</span></li></ol><br><span style="font-weight: bold;">Healthcare<br></span><ol start="1"><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Esteva, Andre, et al. "<a href="https://www.nature.com/articles/nature21056">Dermatologist-level classification of skin cancer with deep neural networks</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">Nature</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"><span>&nbsp;</span>542.7639 (2017): 115-118.</span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"></span><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Choi,
Edward, et al. "Generating Multi-label Discrete Electronic Health
Records using Generative Adversarial Networks." arXiv preprint
arXiv:1703.06490 (2017).</span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Che, Zhengping, et al. "<a href="https://arxiv.org/abs/1606.01865">Recurrent neural networks for multivariate time series with missing values</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">arXiv preprint arXiv:1606.01865</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">(2016).</span><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"></span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Choi, Edward, et al. "<a href="http://proceedings.mlr.press/v56/Choi16.html">Doctor AI: Predicting clinical events via recurrent neural networks</a>."<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">Machine Learning for Healthcare Conference</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">. 2016.</span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"></span><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">Choi, Edward, et al. "<a href="https://dl.acm.org/citation.cfm?id=3098126">GRAM: Graph-based attention model for healthcare representation learning.</a>"<span>&nbsp;</span></span><i style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255);">Proceedings of the 23rd ACM SIGKDD International Conference on Knowledge Discovery and Data Mining</i><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;">. ACM, 2017.</span></li><li><span style="color: rgb(34, 34, 34); font-family: Arial,sans-serif; font-size: 13px; font-style: normal; font-weight: normal; letter-spacing: normal; orphans: 2; text-align: left; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; background-color: rgb(255, 255, 255); display: inline ! important; float: none;"></span><font size="5">Phuoc Nguyen, Truyen Tran, Svetha Venkatesh, <a href="https://arxiv.org/abs/1707.05010"><span style="text-decoration: underline;">Deep Learning to Attend to Risk in ICU</span></a>,&nbsp;&nbsp;<span style="font-style: italic;">IJCAI'17 Workshop on Knowledge Discovery in Healthcare II: Towards Learning Healthcare Systems</span> <span style="font-style: italic;">(KDH 2017</span>)</font><font size="5">.&nbsp;</font></li><li><font size="5">Kien Do, <span style="font-weight: bold;">Truyen Tran</span>, Svetha
Venkatesh.<span style="font-style: italic;"> </span><a href="https://arxiv.org/abs/1703.01454"><span style="text-decoration: underline;">Learning Recurrent Matrix Representation</span></a>, </font><font size="5"><span style="font-style: italic;">Third Representation Learning for Graphs Workshop (ReLiG 2017)</span>, also:<span style="font-style: italic;"> arXiv
preprint arXiv</span>: </font><font size="5"><span style="font-style: italic;">1703.01454</span>.</font></li><li><font size="5">Trang Pham, <span style="font-weight: bold;">Truyen Tran</span>, Dinh
Phung, Svetha Venkatesh, <span style="font-style: italic;">&nbsp;</span><a href="http://www.sciencedirect.com/science/article/pii/S1532046417300710">Predicting healthcare trajectories from medical records: A deep learning approach</a>,</font><font size="5"><span style="font-style: italic;">Journal of Biomedical Informatics</span>, April 2017, DOI: 10.1016/j.jbi.2017.04.001. [<a href="http://arxiv.org/abs/1602.00357">Tech report PDF</a>].</font></li><li><font size="5">Phuoc Nguyen, <span style="font-weight: bold;">Truyen Tran</span>, Nilmini
Wickramasinghe, Svetha Venkatesh, </font><font size="5"><a href="https://arxiv.org/abs/1607.07519"><span style="text-decoration: underline;">&nbsp;Deepr: A Convolutional
Net for Medical Records</span></a>,&nbsp;</font><font size="5"><span style="font-style: italic;">IEEE
Journal of Biomedical and Health Informatics</span>,&nbsp;</font><font size="5">vol. 21, no. 1, pp. 22–30, Jan. 2017, Doi:
10.1109/JBHI.2016.2633963</font></li><li><font size="5">Trang Pham, <span style="font-weight: bold;">Truyen Tran</span>, Dinh
Phung, Svetha Venkatesh, <a href="http://arxiv.org/abs/1602.00357">DeepCare: A Deep
Dynamic Memory Model for Predictive Medicine</a>,&nbsp;<span style="font-style: italic;">PAKDD'16</span>,
Auckland, NZ, April 2016.&nbsp;</font></li><li class="itemize"><font size="5"><span style="font-weight: bold;">Truyen Tran</span>,
Tu
D. Nguyen, D.
Phung, and S. Venkatesh,&nbsp;<span style="font-style: italic;"></span><a href="http://linkinghub.elsevier.com/retrieve/pii/S1532-0464%2815%2900014-3">Learning
vector
representation of medical objects via EMR-driven nonnegative restricted
Boltzmann machines (e-NRBM)</a>,
<span style="font-weight: bold;"></span><span style="font-style: italic;">Journal
of Biomedical Informatics</span>, 2015,
doi:10.1016/j.jbi.2015.01.012.</font></li><li><font size="5">Tu D. Nguyen, <span style="font-weight: bold;">Truyen Tran</span>,
D.
Phung, and S. Venkatesh, <a href="../../../Dropbox%20%28Toby%20Playpad%29/public_html/talks/papers/aaai15_main.pdf">Tensor-variate
Restricted Boltzmann Machines</a>,&nbsp;<span style="font-style: italic;">AAAI</span>
2015.</font></li><li><font size="5">Tu
D. Nguyen, <span style="font-weight: bold;">Truyen Tran</span>,
D. Phung, and S. Venkatesh,&nbsp;<a href="../../../Dropbox%20%28Toby%20Playpad%29/public_html/talks/papers/acml13_nguyen.pdf">Latent
patient profile modelling and
applications with Mixed-Variate Restricted Boltzmann Machine</a>,&nbsp; In
Proc. of <span style="font-style: italic;">17th
Pacific-Asia Conference on Knowledge Discovery and Data Mining
(PAKDD’13)</span>, Gold Coast, Australia, April 2013.</font></li></ol><br><span style="font-weight: bold;"><br>Biomedical NLP<br><br></span><span style="font-weight: bold;">Deep learning fundamental</span><br><ol><li><font size="+1"><a href="https://arxiv.org/abs/1701.07875">Wasserstein GAN</a> (ICML, 2017), a key development of GAN. It's <a href="https://arxiv.org/abs/1704.00028">improved version</a> (ICML, 2017).</font></li><li><font size="+1"><a href="http://papers.nips.cc/paper/5423-generative-adversarial">Generative Adversarial Nets</a>&nbsp;(NIPS, 2014), a totally new way of generating high quality data</font></li><li><font size="+1"><a href="http://arxiv.org/abs/1312.6114">Variational Auto-Encoders</a> (NIPS, 2014), a new powerful way to learn a generative model</font></li></ol><ol><li value="2"><font size="+1"><a href="http://www.nature.com/nature/journal/vaop/ncurrent/pdf/nature20101.pdf">Hybrid computing using a neural network with dynamic external memory</a> (DeepMind, 2016), role of external memory. Early version: </font><font size="+1"><a href="http://arxiv.org/abs/1410.5401">Neural Turing machine</a> (2014).</font></li><li><font size="+1"><a href="https://arxiv.org/abs/1506.07285">Ask me anything: dynamic memory networks </a>(ICML, 2016), a cool way to do question answering using memory</font></li><li><font size="+1"><a href="http://papers.nips.cc/paper/5846-end-to-end-memory-networks">End-to-end memory networks</a> (NIPS, 2015), role of memory</font></li></ol><br></td></tr></tbody></table><br><p style="" align="justify"><br>
&nbsp;</p>
</body></html>